migrations/20201102184120_lnet.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS lnet (
+    id                  serial PRIMARY KEY,
+    host_id             int NOT NULL UNIQUE,
+    state               text NOT NULL,
+    nids                int[] NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS nid (
+    id serial PRIMARY KEY,
+    net_type text NOT NULL,
+    host_id int NOT NULL UNIQUE,
```

Comment:
Should this really be unique?

Reply:
I thought when we discussed earlier we said it should be unique. Any reason why it wouldn't be?

Reply:
because then you could only have one host_id regardless of how many nids were on that host.